### PR TITLE
Document party menu

### PR DESCRIPTION
**Pull request by cbt6**

### CODE REVIEW

File: src/party_menu.c [hunk 1]
```diff
@@ -1788,25 +1767,24 @@ u8 GetMonAilment(struct Pokemon *mon)
     return AILMENT_NONE;
 }
 
+#define minigameBitflag data[0]
```
Comment (Kurausukun) [inline src/party_menu.c:1770]: I think we usually prefix with t for these kinds of definitions, e.g. `tMingameBitflag`

Comment (cbt6) [reply src/party_menu.c:1770]: I assumed the convention was to use `t` prefix for task data (just like the `s` prefix for sprite data)?

File: src/party_menu.c [hunk 2]
```diff
@@ -83,6 +83,9 @@
 #define MENU_DIR_RIGHT    2
 #define MENU_DIR_LEFT    -2
 
+#define SLOT_CONFIRM PARTY_SIZE
+#define SLOT_CANCEL  PARTY_SIZE + 1
```
Comment (GriffinRichards) [inline src/party_menu.c:87]: Parentheses

File: src/party_menu.c [hunk 3]
```diff
@@ -1200,8 +1207,6 @@ static void HandleChooseMonSelection(u8 taskId, s8 *slotPtr)
                 TryEnterMonForMinigame(taskId, (u8)*slotPtr);
             break;
         default:
-        case PARTY_ACTION_ABILITY_PREVENTS:
-        case PARTY_ACTION_SWITCHING:
             PlaySE(SE_SELECT);
```
Comment (GriffinRichards) [inline src/party_menu.c:1210]: Leave these if it's possible for `gPartyMenu.action` to be either value here, to help people searching for where they might be read.

File: src/party_menu.c (hunk 1 above)
Comment (GriffinRichards) [reply src/party_menu.c:1770]: Yeah `t` is meant to be short for `task`. We don't have any prefix for generic struct data arrays, this is fine with me.

**Pull request merged**